ext/phashion_ext/extconf.rb

```diff
@@ -40,7 +41,7 @@
     system("cp -f libpHash.a libpHash_gem.a")
     system("cp -f libpHash.la libpHash_gem.la")
   end
-  $LIBS = " -lpthread -lpHash_gem -lstdc++ -ljpeg -lpng -lmvec"
+  $LIBS = " -lpthread -lpHash_gem -lstdc++ -ljpeg -lpng -lm"
```

Comment:
I'm not 100% sure, but could we test if it works with svgs?
`lm` - this is the math lib
`lmvec` - this one for vectorized algebra

I also tried to google is there any difference between them and looks like there is and i'm not sure, but let's test it with svgs before we deploy please? 🙏 

`
In summary, lm is used for fitting linear regression models, while lmvec could potentially refer to a library or function that performs vectorized linear algebra operations. If you have more specific information about the lmvec library or function, please provide additional details for a more accurate answer.
`

Reply:
@gate8team Sure! I'll fetch the latest master of `phashion` locally and try to calculate the phash for a SVG with it

Reply:
Thanks 🙏

Reply:
@gate8team With a little delay, but just confirmed the current version of phashion in Artisan calculates the same phash value as the master of the original repo. I think we're good to proceed but you raised a valid point - in the least signal of trouble we add `-lmvec` back in.

![image](https://github.com/printbear/phashion/assets/3844400/1fa64c01-a5a8-496e-b9f8-b08076e966a3)